Dockerfile.test

```
@@ -0,0 +1,8 @@
+FROM        golang:1.14.3
+
+ENV         GO111MODULE=on
+
+COPY    . ./
+
+ENTRYPOINT  ["make"]
```

This will probably fail.
Try adding this:
```suggestion
RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env 
GOPATH)/bin v1.24.0
RUN go get -u github.com/onsi/ginkgo/ginkgo@v1.12.2 \
              golang.org/x/tools/cmd/goimports@v0.0.0-20200520220537-cf2d1e09c845 \
              github.com/golang/mock/mockgen@v1.4.3 \
              github.com/vektra/mockery/.../@v1.1.2
ENTRYPOINT  ["make"]
```

---

thanks! still failing though.

---

link to the failure?

---

https://paste.centos.org/view/2b2eff08

---

solved